tests/acceptance/features/webUIRestrictSharing/restrictSharing.feature

```diff
@@ -27,20 +27,20 @@ Feature: restrict Sharing
     And user "Brian" has uploaded file "lorem.txt" to "simple-folder/lorem.txt" in the server
     And user "Brian" has logged in using the webUI
 
-  @smokeTest
+  @smokeTest @skipOnOCIS
   Scenario: Restrict users to only share with users in their groups
+#   shares-only-with-groups-members configuration is not available now in the oCIS webUI
```

Comment:
Can we skip this whole feature file? The features mostly test the admin-sharing-settings (uses `occ`)
```suggestion
```

Reply:
I also thought the same while modifying this file. But this scenario passes because by default  `shareapi_only_share_with_membership_groups` is off currently in ocis. From my side solution is either we skip the whole scenario doesn't matter if it passes or fails. Another solution, let the passing scenario run. 
```feature

  Scenario: Do not restrict users to only share with groups they are member of
    Given the setting "shareapi_only_share_with_membership_groups" of app "core" has been set to "no" in the server
    When the user shares folder "simple-folder" with group "grp2" as "Viewer" using the webUI
    And user "Carol" accepts the share "Shares/simple-folder" offered by user "Brian" using the sharing API in the server
    Then as "Carol" folder "/Shares/simple-folder" should exist in the server
```

Reply:
This scenario should be covered somewhere for oCIS too. Looks like it is just to share and accept the share (so there might be whole lot of tests that covers it).
As I mention, the main purpose here is to test admin setting. So, maybe we can skip this also.

CC @phil-davis